utils/checkDiamondStorage.js

```diff
@@ -0,0 +1,23 @@
+import { ethers } from "hardhat";
```

Comment:
Please import `ethers` from `ethers` because we should not be using hardhat anywhere in our codebase anymore.